requirements.txt

```diff
@@ -0,0 +1,14 @@
+python-dotenv>=1.0.0,<1.1.0
```

Comment:
Remove requirements.txt and move dependencies under `#Dev` and `#Build` to pyproject.toml